tools/vagrant/simple/hello.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+vagrant up
+
+vagrant ssh -c "sudo apt-get install -y git"
+
+vagrant ssh -c "git clone --depth=1 https://github.com/openwhisk/openwhisk.git"
+
+vagrant ssh -c "cd openwhisk/tools/ubuntu-setup && source all.sh"
+
+vagrant reload
+
+vagrant ssh -c "cd openwhisk && tools/db/couchdb/start-couchdb-box.sh whisk_admin some_passw0rd"
```

Comment:
That's a heavy weight way of generating the config file, no? Maybe that can be done more directly.